api/src/providers/githubProvider.ts

```diff
@@ -37,3 +39,25 @@ export async function getAuditors() {
 
   return response;
 }
+
+export async function getProviderRegions() {
```

Comment:
Probably best to move this to another provider in `db/` since it's not querying data from github